lib/util.js

```diff
@@ -791,25 +783,29 @@ function formatProperty(ctx, value, recurseTimes, visibleKeys, key, array) {
     if (array && key.match(/^\d+$/)) {
       return str;
     }
-    name = JSON.stringify('' + key);
+    name = `"${key}"`;
```

Comment:
That's right, I forgot about the escaping:
```
var a = ' " ';
`"${a}"` // " " "
JSON.stringify(a) // " \" "
```

Sorry for the wrong suggestion !